app/src/containers/Map/DrawingManager.js

```diff
@@ -1,14 +1,15 @@
 import { connect } from 'react-redux';
 import DrawingManager from 'components/Map/DrawingManager';
-import { saveCoordinates } from 'actions/areas';
+import { saveEditingArea } from 'actions/areas';
 
 const mapStateToProps = state => ({
-  map: state.map.googleMaps
+  map: state.map.googleMaps,
+  polygonColor: state.areas.editingArea.color,
```

Comment:
linter is complaining here. use the git hooks to validate before committing - more info Monday